Languages/update-translated-strings.sh

```diff
@@ -5,5 +5,6 @@
 # script before committing changes to the .po files, otherwise the .po
 # files might get modified the next time you try to build using cmake.
 
+cd ${0/update-translated-strings.sh/}/..
```

Comment:
I think you can use `dirname $0` here, which would make the script work even if it gets renamed (and avoid breakage if someone forgets to change this line after renaming, or makes a typo).